contracts/prebuilts/account/utils/AccountCore.sol

```diff
@@ -189,6 +189,11 @@ contract AccountCore is IAccountCore, Initializable, Multicall, BaseAccount, Acc
                         Internal functions
     //////////////////////////////////////////////////////////////*/
 
+    /// @dev Returns the salt used when deploying an Account.
+    function _generateSalt(address _admin, bytes memory _data) internal view virtual returns (bytes32) {
+        return keccak256(abi.encodePacked(_admin, _data));
```

Comment:
I think we should use encode instead of encode packed here ? From what I read online it's better suited for dynamic types like bytes